tests/functional/partial_parsing/test_file_diff.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from dbt.tests.util import run_dbt, write_artifact
+
+first_file_diff = {
+    "deleted": [],
+    "changed": [],
+    "added": [{"path": "models/model_one.sql", "content": "select 1 as fun"}],
+}
+
+
+second_file_diff = {
+    "deleted": [],
+    "changed": [],
+    "added": [{"path": "models/model_two.sql", "content": "select 123 as notfun"}],
+}
+
+
+class TestFileDiffs:
```

Comment:
Since we plan to use manifest as some sort of interface between parsing/execution, should we test the parsing based on checking the manifest instead of running dbt?